src/abstractdataframe/io.jl

```diff
@@ -133,6 +143,7 @@ function Base.show(io::IO, ::MIME"text/html", df::AbstractDataFrame)
     end
     write(io, "</tbody>")
     write(io, "</table>")
+    write(io, "<p>$(digitsep(n)) rows x $(digitsep(ncol(df))  ) columns</p>")
```

Comment:
Fair enought- I just followed the pandas example, but it does seem more useful to get this info right at the top rather than having to scroll down for it!

Reply:
Somehow managed to create two pull requests which achieve the same thing, but this should be sorted now - apologies for the delay!